tests/fixtures/SuppressASTLinterAwait/await_in_while.php.in

```diff
@@ -18,9 +18,9 @@ async function spin_forever_async(
   }
 }
 
-// This one should fail since the FIXME is outside the loop.
+// This one should fail since the FIXME is outside the scope.
+/* HHAST_FIXME[DontAwaitInALoop] */
 async function spin_forever_async_fail(
 ): Awaitable<void> {
-  /* HHAST_FIXME[DontAwaitInALoop] */
```

Comment:
Old behavior seems correct - the lint error isn't a problem with the loop, but the await statement - tying it to the while seems bad.